insights/specs/insights_archive.py

```diff
@@ -192,6 +192,7 @@ class InsightsArchiveSpecs(Specs):
     ovs_vsctl_show = simple_file("insights_commands/ovs-vsctl_show")
     parted__l = simple_file("insights_commands/parted_-l_-s")
     passenger_status = simple_file("insights_commands/passenger-status")
+    pci_rport_target_disk_paths = simple_file("insights_commands/find_pci_rport_target_disk_paths")
```

Comment:
The `path` of the simpe_file is incorrect, please use the `mangle` command to generate it based on the "simple_command" in defult.py